compiler/qsc_rca/src/core.rs

```diff
@@ -1927,10 +1927,27 @@ impl<'a> Visitor<'a> for Analyzer<'a> {
             ItemKind::Callable(decl) => {
                 self.visit_callable_decl(decl);
             }
+            ItemKind::Export(
+                _,
+                qsc_fir::fir::ItemId {
+                    package: Some(package),
+                    item,
+                },
+            ) => {
+                let package = self.package_store.get(*package);
+                let Some(item) = package.items.get(*item) else {
+                    return;
+                };
```

Comment:
I actually think it makes sense to turn this into an `.expect` since an item should be found in the package it's listed with. That would help catch any (admittedly unlikely) bugs.